Cargo.toml

```diff
@@ -15,10 +15,16 @@ members = [
 
 exclude = ["fuzz"]
 
-[patch.crates-io]
-# tokio = { path = "../tokio/tokio" }
-# mio = { git = "https://github.com/tokio-rs/mio.git" }
-# h2 = { git = "https://github.com/hyperium/h2.git" }
+[workspace.package]
+version = "0.23.0-alpha.1"
+authors = ["Benjamin Fry <benjaminfry@me.com>"]
```

Comment:
RFC 3052 recommends leaving the authors out of the package metadata, so that's what I've been doing for all of the other crates I maintain.

Reply:
Oh interesting. Ok. I can strip that out.